src/main/java/org/veupathdb/eda/binaryfiles/dumper/Main.java

```diff
@@ -64,4 +74,32 @@ public static void main(String[] args) throws Exception {
       studyDumper.dumpStudy();
     }
   }
+
+  public static Map<String, String> parseConfig(Path filePath) {
```

Comment:
If gus.config takes the format of a Properties file, you can parse it using `Properties prop = new Properties(); prop.load(inputStream)`